client/src/components/CountdownCircle/CountdownCircle.stories.js

```diff
@@ -0,0 +1,22 @@
+import _ from "lodash";
+import React from "react";
+
+import { CountdownCircle } from "./CountdownCircle.js";
+
+export default {
+  title: "CountdownCircle",
+  component: CountdownCircle,
+};
+
+// not sure if the component works or not
+const Template = (args) => <CountdownCircle {...args} />;
```

Comment:
Should if that `Countdown` bug is fixed.

Reply:
Fixed!